presto-main/src/main/java/com/facebook/presto/server/ServerConfig.java

```diff
@@ -21,7 +21,7 @@
 public class ServerConfig
 {
     private boolean coordinator = true;
-    private String prestoVersion;
+    private String prestoVersion = getClass().getPackage().getImplementationVersion();
```

Comment:
can you add a check in `setPrestoVersion` that parameters is not null?

Reply:
We usually don't do that. Instead, we use `@NotNull` and this is what i did.

Reply:
Correct, config classes are mutable beans that use the bean validation framework, so they need to be able to store and return invalid values (that the validation framework can validate as a group).